book/src/dev/release-process.md

```diff
@@ -0,0 +1,118 @@
+We strive to ensure that future changes are always introduced in a predictable way.
+We want everyone who depends on Zebra to know when and how new features are added, and to be well-prepared when obsolete ones are removed.
+
+<a id="versioning"></a>
+
+## Zebra versioning
+
+Zebra version numbers indicate the level of changes that are introduced by the release.
+This use of [semantic versioning](https://semver.org/ "Semantic Versioning Specification") helps you understand the potential impact of updating to a new version.
+
+Zebra version numbers have three parts: `major.minor.patch`.
+For example, version `3.1.11` indicates major version 3, minor version 1, and patch level 11.
+
+The version number is incremented based on the level of change included in the release.
+
+<div class="alert pre-release">
+
+**NOTE**: <br />
+As Zebra is in a `pre-release` state (is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version).
+The pre-release version is denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version.
+
+</div>
+
+| Level of change | Details |
+|:---             |:---     |
+| Major release   | Contains significant new features, and commonly correspond to network upgrades; some technical assistance may be needed during the update. When updating to a major release, you may need to follow the specific upgrade instructions provided in the release notes. |
+| Minor release   | Contains new smaller features. Minor releases are should be fully backward-compatible. No technical assistance is expected during update.  If you want to use the new features in a minor release, you might need to follow the instructions in the release notes. |
+| Patch release   | Low risk, bug fix release. No technical assistance is expected during update. |
+
+<a id="supported-releases"></a>
+
+### Supported Releases
+
+Older Zebra versions are always supported until:
+* the next Zebra major, minor or patch release.
+
+Initially, we can only guarantee support for the latest Zebra release. We might be able to provide support for earlier releases, or we might ask you to upgrade to the latest release.
+
+Our support periods will be extended as we gain experience with supporting Zebra releases.
+
+Older Zebra versions that only support previous network upgrades will never be supported, because they are operating on an unsupported Zcash chain fork.
+
+<a id="updating"></a>
+
+### Supported update paths
+
+You can update to any version of Zebra, provided that the following criteria are met:
+
+* The version you want to update *to* is supported.
+* The version you want to update *from* is within one major version of the version you want to
+    upgrade to.
+
+See [Keeping Up-to-Date](guide/updating "Updating your projects") for more information about updating your Zebra projects to the most recent version.
+
+<a id="previews"></a>
+
+### Preview releases
```

Comment:
I think this answers my comment above about giving an example of pre-release versioning, maybe we can link to here from there instead. e.g for an example see "Preview releases"